dialects/math/math.thorin

```diff
@@ -153,6 +154,25 @@
 ///
 .ax %math.gamma(t, l): Π.[pe: «2; .Nat»][.Nat][%math.F pe] -> %math.F pe, normalize_gamma;
 ///
+/// ### %math.abs
+///
+/// [Absolute value] (https://en.wikipedia.org/wiki/Absolute_value) Returns absolute value of the provided value
+///
+.ax %math.abs: Π.[pe: «2; .Nat»][.Nat][%math.F pe] -> %math.F pe, normalize_abs;
+///
+/// ### %math.round
+///
+/// [Rounding down], [Rounding up], [Rounding to next integer] and [Rounding towards zero/Truncating] (https://en.wikipedia.org/wiki/Rounding) 
```

Comment:
`[Text](Link)` is the markdown tag for a link.
Example: `[google](https://google.com)` becomes [google](https://google.com)

This comment probably does not render as expected.

Suggestion:
``` 
Common rounding operations on floating points:
- [Rounding down](https://en.wikipedia.org/wiki/Rounding#Rounding_down) (`f`loor)
- ...
````